packages/core/src/credentialsV1/common.ts

```diff
@@ -10,15 +10,17 @@ import { base58Decode, base58Encode } from '@polkadot/util-crypto'
 import { hexToU8a } from '@polkadot/util'
 
 import type { HexString } from '@kiltprotocol/types'
+import { Caip19, Caip2, SDKErrors } from '@kiltprotocol/utils'
 
-import { Caip19, Caip2 } from './CAIP/index.js'
-import {
-  KILT_ATTESTER_DELEGATION_V1_TYPE,
-  KILT_CREDENTIAL_IRI_PREFIX,
-} from './constants.js'
-import { CredentialMalformedError } from './errors.js'
 import type { KiltAttesterDelegationV1, KiltCredentialV1 } from './types.js'
 
+export const spiritnetGenesisHash = hexToU8a(
+  '0x411f057b9107718c9624d6aa4a3f23c1653898297f3d4d529d9bb6511a39dd21'
+)
+export const KILT_ATTESTER_LEGITIMATION_V1_TYPE = 'KiltAttesterLegitimationV1'
```

Comment:
wanted to expose them under the KiltCredentialV1 namespace but was afraid circular dependencies may break things, so they are defined here and exposed there